.github/actions/docker-build-and-push/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Docker Build & Push
+description: Composite GitHub Action to build and push Docker images to the DLCS GitHub Packages repositories.
+
+inputs:
+  image-name:
+    description: "Name of the image to push to the GHCR repository."
+    required: true
+  dockerfile:
+    description: "The Dockerfile to build and push."
+    required: true
+  context:
+    description: "The context to use when building the Dockerfile."
+    required: true
+  github-token:
+    description: "The GitHub token used when interacting with GCHR."
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - id: checkout
+      uses: actions/checkout@v2
+    - id: docker-setup-buildx
+      uses: docker/setup-buildx-action@v2
+      with:
+        driver-opts: |
+          image=moby/buildkit:v0.10.6
```

Comment:
I'm not sure if this is required any longer, I'll take a note to test removing from Protagonist. It was related to a ghcr/buildkit issue from a few months ago (https://github.com/docker/build-push-action/issues/761#issuecomment-1398918693)